osmosis/assetlist.json

```diff
@@ -14280,6 +14280,30 @@
       "logo_URIs": {
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/BRNZ.svg"
       }
+    },
+    {
+      "description": "OnE mEmEcOiN tO cOnNeCt oL ImBeCiles - aNd in Da Cosmos BiNd DeM",
+      "denom_units": [
+        {
+          "denom": "factory/osmo1kqdw6pvn0xww6tyfv2sqvkkencdz0qw406x54r/IBC",
```

Comment:
The base denom unit must always be exponent 0, so this definition must be incorrect. Please define two denom units, one for the base (exponent 0), and one for desired display (likely exponent 6), and reference that display denom unit via the 'display' field.

Reply:
Updated this like so: 
<img width="575" alt="image" src="https://github.com/cosmos/chain-registry/assets/164512747/6e4c1638-45b3-419a-98c5-c0caab8539e1">

Does this new "display" denom need to be registered onchain in someway? As I cant find any metadata update function available in osmosis tokenfactory module

Reply:
There is a metadata update msg in the osmosis tokenfactory module

https://github.com/osmosis-labs/osmosis/blob/main/proto/osmosis/tokenfactory/v1beta1/tx.proto#L119